lib/rbs_rails/cli.rb

```diff
@@ -0,0 +1,166 @@
+require "optparse"
+require "rbs_rails/cli/configuration"
+
+module RbsRails
+  # @rbs &block: (CLI::Configuration) -> void
+  def self.configure(&block) #: void
+    CLI::Configuration.configure(&block)
+  end
+
+  class CLI
+    attr_reader :config_file #: String?
+
+    # @rbs argv: Array[String]
+    def run(argv) #: Integer
+      parser = create_option_parser
+
+      begin
+        args = parser.parse(argv)
+        subcommand = args.shift || "help"
+
+        case subcommand
+        when "help"
+          $stdout.puts parser.help
+          0
+        when "version"
+          $stdout.puts "rbs_rails #{RbsRails::VERSION}"
+          0
+        when "all"
+          load_application
+          load_config
+          generate_models
+          generate_path_helpers
+          0
+        when "models"
+          load_application
+          load_config
+          generate_models
+          0
+        when "path_helpers"
+          load_application
+          load_config
+          generate_path_helpers
+          0
+        else
+          $stdout.puts "Unknown command: #{subcommand}"
+          $stdout.puts parser.help
+          1
+        end
+      rescue OptionParser::InvalidOption => e
+        $stderr.puts "Error: #{e.message}"
+        $stdout.puts parser.help
+        1
+      end
+    rescue StandardError => e
+      $stderr.puts "Error: #{e.message}"
+      1
+    end
+
+    private
+
+    def config #: Configuration
+      Configuration.instance
+    end
+
+    def load_config #: void
+      if config_file
+        load config_file
+      else
+        if File.exist?(".rbs_rails.rb")
+          load ".rbs_rails.rb"
+        elsif Rails.root.join("config/rbs_rails.rb").exist?
+          load Rails.root.join("config/rbs_rails.rb").to_s
+        end
+      end
+    end
+
+    def load_application #: void
+      require_relative "#{Dir.getwd}/config/application"
+
+      install_hooks
+
+      Rails.application.initialize!
+    end
```

Comment:
I found `require_relative` raises LoadError, and it will not be caught on `rescue StandardError`. It's better to handle it here.